src/main/java/org/humancellatlas/ingest/process/Process.java

```diff
@@ -69,4 +69,8 @@ public void addProtocol(Protocol protocol) {
         protocols.add(protocol);
     }
 
+    public void removeProtocol(Protocol protocol) {
+        protocols.remove(protocol);
```

Comment:
If you would like to build a fluent API then you can do it as you did in the `src/main/java/org/humancellatlas/ingest/file/File.java` and add `return this;` to the end of the method

```suggestion
        protocols.remove(protocol);
        return this;
```